nucypher/characters/lawful.py

```diff
@@ -1106,11 +1103,8 @@ def run(self,
             worker: bool = True,
             pruning: bool = True,
             interactive: bool = False,
-            prometheus: bool = False,
             start_reactor: bool = True,
-            metrics_port: int = None,
-            metrics_prefix: str = 'ursula',
-            metrics_listen_address: str = ''
+            prometheus_config: PrometheusMetricsConfig = None,
```

Comment:
Hmm - this needs a second look.  all of the other inputs to this function are feature switches as booleans.  Is this a special case?

Reply:
True.  The configuration needs to happen at an earlier stage to be consistent.